locales/fr_FR/pcsx2_Iconized.po

```diff
@@ -877,225 +883,7 @@ msgid ""
 "not a critical error, since the sVU rec is obsolete, and you should use "
 "microVU instead anyway. :)"
 msgstr ""
-"Out of Memory (ou presque) : le recompilateur SuperVU n'est pas parvu à "
+"Out of Memory (ou presque) : le recompilateur SuperVU n'est pas parvenu à "
 "allouer suffisamment de mémoire virtuelle, ce qui rend impossible son "
-"utilisation. Ce n'est pas une erreur critique, puisque le recompilateur sVU "
-"est obsolète et puisque vous devriez utiliser microVU ! :p"
```

Comment:
Removed a bit too much? Line 882-884. 
Also Warning that endline breaks.
![image](https://user-images.githubusercontent.com/24227051/82128248-f8f87d80-97b9-11ea-8aa5-b87e94f24435.png)

Reply:
Probably just GitHub being a bit lazy here, removed nothing more than the old translations.

Reply:
It means that the last characters in the file is not a newline sequence.